src/AzureExtension/DevBox/Logging.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHome.Logging;
+using Windows.Storage;
+
+namespace AzureExtension.DevBox;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1649:File name should match first type name", Justification = "Logging project is shared")]
+public class Log
+{
+    private static Logger? _logger;
+
+    public static Logger? Logger()
+    {
+        try
+        {
+            _logger ??= new Logger("DevBox", GetLoggingOptions());
+        }
+        catch
+        {
+            // Do nothing if logger fails.
+        }
+
+        return _logger;
+    }
+
+    public static void Attach(Logger logger)
+    {
+        if (logger is not null)
+        {
+            _logger?.Dispose();
+            _logger = logger;
+        }
+    }
+
+    public static Options GetLoggingOptions()
+    {
+        return new Options
+        {
+            LogFileFolderRoot = ApplicationData.Current.TemporaryFolder.Path,
+            LogFileName = "DevBox_{now}.log",

```

Comment:
Dev Home recently changed its log extension to `.dhlog`.

Reply:
sure will do